test/e2e/specs/onboarding/signup__with-theme-LOHP.ts

```diff
@@ -74,19 +74,17 @@ describe( 'Lifecyle: Logged Out Home Page, signup, onboard, launch and cancel su
 			if ( calypsoUrl.hostname !== 'wordpress.com' ) {
 				// Reroute the click to the current Calypso URL.
 				await page.route( themeButtonUrl.href, async ( route ) => {
-					themeButtonUrl.host = calypsoUrl.host;
-					themeButtonUrl.protocol = calypsoUrl.protocol;
-
 					await route.abort();
-					await page.goto( themeButtonUrl.href );
```

Comment:
A `goto` right after `abort` will obviously break things. Moving it out of the `route()` completely.